CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# Optimizely Csharp SDK Changelog
+
+## 3.3.0
+September 19th, 2019
+
+### New Features:
+- Configuration manager is set to PollingProjectConfigManager and polling will be started by default. 
+- Added support for event batching via the event processor.
+- Events generated by methods like `Activate`, `Track`, and `IsFeatureEnabled` will be held in a queue until the configured batch size is reached, or the configured flush interval has elapsed. Then, they will be combined into a request and sent to the event dispatcher.
+- To configure event batching, set the `MaxEventBatchSize` and `MaxEventFlushInterval` properties in the `OptimizelyFactory` using `OptimizelyFactory.SetBatchSize(int batchSize)` and `OptimizelyFactory.SetFlushInterval(TimeSpan flushInterval)` and then create `OptimizelyFactory.NewDefaultInstance`.
+- Event batching is enabled by default. `eventBatchSize` defaults to `10`. `eventFlushInterval` defaults to `30000` Milliseconds.
+- Updated the `Dispose` method representing the process of closing the instance. When `Dispose` is called, any events waiting to be sent as part of a batched event request will be immediately batched and sent to the event dispatcher.
+- If any such requests were sent to the event dispatcher, `Stop` waits for provided `TimeoutInterval` before stoping, so that events get successfully dispatched.
+- `OptimizelyFactory` now provides support of setting configuration variables from ***App.config***, User will now be able to provide configuration variables of `HttpConfigManager` and `BatchEventProcessor` in ***App.config***. Steps of usage are provided in [README.md](https://github.com/optimizely/csharp-sdk#using-appconfig-in-optimizelyfactory).
+
+### Deprecated
+- `EventBuilder` was deprecated and now we will be using `UserEventFactory` and `EventFactory` to create LogEvent Object.
+- Deprecated `Activate` and `Track` notifications in favor of explicit `LogEvent` notification.
```

Comment:
We should only mention `Track` notification listener here as `Activate` was already marked as deprecated previously.